services/core-web/common/reducers/permitReducer.js

```diff
@@ -39,7 +39,7 @@ const permitReducerObject = {
   [PERMITS]: permitReducer,
 };
 
-export const getPermits = (state) => state[PERMITS].permits;
+export const getUnformattedPermits = (state) => state[PERMITS].permits;
```

Comment:
What is meant by "unformatted permits"?

Reply:
I had to update the selector for 'getPermits' to format the site_properties into a structure that the BE accepts, 

I could have created a new selector and updated the components to read that selector, but that seemed like more effort than to rename the reducer export as the "unformattedPermits" and create a new selected called "getPermits" that all the components are already referencing.